src/book/book.service.ts

```diff
@@ -37,20 +37,8 @@ export class BookService {
   }
 
   async getAllBooks(page: number = 1, limit: number = 20, filter: BookFilterDto = {}) {
-    const filterConditions = getFilterConditions(filter)
     const sortConditions = getSortConditions(filter)
-
-    const where = {
-      OR: [
-        filterConditions,
-        {
-          isbn: {
-            contains: filter.title,
-            mode: 'insensitive',
-          },
-        },
```

Comment:
[?] This is not necessary anymore? I mean, if this is removed it doesn't break anything?

Reply:
This would be good to have as a feature because it ables to search by isbn using the same search text. The only problem is that if someone makes an empty search, and wanted to filter by something else (for example by maxPrice) then the later filter would not apply because title="" essentially would return all books (this is because is inside an OR). You can see it right now in prod with the following link:

https://snuffleevees-paltanchez.github.io/search?title=&maxPrice=1000

You would see that the filter of max price is not being applied. This is why I removed it.